### ISSUE TITLE
Link s3 uploads to standards + university

### ISSUE THREAD

**Issue by AmjadShubbar** [bahrain-bp/bqa-genai-challenge#36]

**Comment by maalsaegh** [comment]
@maalsaegh

**Comment by AmjadShubbar** [comment]
@maalsaegh @cerashdan 
I am facing issues completing this task. I have completed the (Linking to university + linking to standards/indicators) using metadata + Retrieval of all s3 files with their metadata and mapping them together. However, I am facing issues when trying to filter the results using the metadata key pair createdBy. I think the issue is related with my mapping because it has nested json results but I'm not sure. 
There was another problem because I am using url parameter to retrieve the username and it seems that url default encoding ignores case sensitivity but I solved it and the passed username matches the provided in the url parameter.
I tried searching online help but did not find any helpful articles about metadata filtering.




![Image](https://github.com/bahrain-bp/bqa-genai-challenge/assets/120450558/a6854bf3-4695-474b-aece-7eff853d99f2)
